backend/index.js

```diff
@@ -2,15 +2,19 @@ const app = require("express")();
 const server = require("http").createServer(app);
 const io = require("socket.io")(server, {
     // Set up of CORS settings for socket.io server
+    // Reason for all site access is for the ease of development, since we might have various local/cloud website setup for testing purposes.
+    // While allowing all origins may pose a security thread for a banking website, as any site can make a request using the same session
+    // This won't be a problem in our website, as there are no local session/cookies returned to the user. 
+    // Please refer to the post: https://stackoverflow.com/questions/12001269/what-are-the-security-risks-of-setting-access-control-allow-origin
     cors: {
-        origin: [
-            "http://localhost:3000"
-        ],
-        methods: ["GET", "POST"]
+        origin: '*', // Allow all origin to connect to the website
```

Comment:
I think we discussed in a meeting that we could move the CORS settings to an external "config" file? I think it could help to have that separation from the configuration and the server code.